src/tests/Microsoft.PowerFx.Interpreter.Tests/SerializationTests.cs

```diff
@@ -32,7 +33,25 @@ public void DateTimeSerializationTests()
 
                     Assert.Equal(dateTimeValue.Value, dateTimeValueDeserialized.Value);
                 }
-            }            
-        }            
+            }
+        }
+
+        [Fact]
+        public void TempTests()
+        {
+            var engine = new RecalcEngine(new PowerFxConfig());
+
+            var expression = "If(true,";
+
+            var ptBR_opt = new ParserOptions() { Culture = new CultureInfo("pt-BR") };
+            var frFR_opt = new ParserOptions() { Culture = new CultureInfo("fr-FR") };
+            var enGB_opt = new ParserOptions() { Culture = new CultureInfo("en-GB") };
+
+            var ptBR_check = engine.Check(expression, options: ptBR_opt);
```

Comment:
>ptBR_check [](http://example.com/codeflow?start=16&length=10)

Can we add a test that looks at the error message in the various locales, and verifies it's actually translated. 

That's the most critical customer-facing part.

Reply:
These are temporary tests. The real ones are in the src/tests/Microsoft.PowerFx.Core.Tests/ParseTests.cs file.